examples/dumpframes/ebpf/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:8
+
+RUN dnf install -y make cmake gcc gdb clang llvm clang-devel llvm-devel autoconf libtool kernel-devel libbpf elfutils-libelf-devel elfutils-devel
+
+RUN /bin/bash -c " \
+    curl -fsSL https://golang.org/dl/go1.16.2.linux-amd64.tar.gz -o /tmp/golang.tar.gz && \
+    tar -C /usr/local -xzf /tmp/golang.tar.gz"
+
+ENV PATH=$PATH:/usr/local/go/bin:/root/go/bin/
+
+RUN dnf -y install git
+
+RUN go get github.com/cilium/ebpf/cmd/bpf2go
```

Comment:
I tried regenerating and the generated `ipproto_bpfeb.go` and `ipproto_bpfel.go` were different with an incompatible API - I think it's because it got changed in `bpf2go` just a few days ago: https://github.com/cilium/ebpf/commit/18b642a82f9f7c15874e8eb6c051752d3024ee8c

A simple fix could be to explicitly specify the version of `bpf2go` to use and set it to the previous version v0.3.0, then it works fine:
```
diff --git a/examples/dumpframes/ebpf/Dockerfile b/examples/dumpframes/ebpf/Dockerfile
index 9120ede..6386b44 100644
--- a/examples/dumpframes/ebpf/Dockerfile
+++ b/examples/dumpframes/ebpf/Dockerfile
@@ -10,6 +10,6 @@ ENV PATH=$PATH:/usr/local/go/bin:/root/go/bin/
 
 RUN dnf -y install git
 
-RUN go get github.com/cilium/ebpf/cmd/bpf2go
+RUN go get github.com/cilium/ebpf/cmd/bpf2go@v0.3.0
 
 ENTRYPOINT ["bash", "-c", "cd /target/examples/dumpframes/ebpf && go generate"]
```

Reply:
That makes sense. I'll make that change, and probably update to the latest.